src/main/java/hudson/plugins/sonar/SonarInstallation.java

```diff
@@ -57,6 +63,39 @@
     return sonarDescriptor.getInstallations();
   }
 
+  public static final SonarInstallation[] enabled() {
+    Jenkins jenkins = Jenkins.getInstance();
+    if (jenkins == null) {
+      // for unit test
+      return new SonarInstallation[0];
+    }
+    SonarPublisher.DescriptorImpl sonarDescriptor = Jenkins.getInstance().getDescriptorByType(SonarPublisher.DescriptorImpl.class);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'getInstance' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS2259)